server/index.js

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
This removes all the typing info - this shouldn't be necessary?

Reply:
I put that in temporarily while I was trying things out. Forgot to remove it. 
Done